emails/views.py

```diff
@@ -249,15 +249,11 @@ def _sns_message(message_json):
 
     from_address = parseaddr(mail['commonHeaders']['from'])[1]
     subject = mail['commonHeaders']['subject']
-    email_message = message_from_string(message_json['content'])
-    for message_payload in email_message.get_payload():
-        # TODO: check Content-Transfer-Encoding to see if additional decoding
-        # is needed.
-        # E.g., b64decode(message_payload.get_payload()).decode('utf-8')
-        if message_payload.get_content_type() == 'text/plain':
-            text_content = message_payload.get_payload()
-        if message_payload.get_content_type() == 'text/html':
-            html_content = message_payload.get_payload()
+    email_message = message_from_string(
+        message_json['content'], policy=policy.default
+    )
+
+    text_content, html_content = _get_text_and_html_content(email_message)
```

Comment:
Note: we should probably check `email_message.is_multipart()` before assuming the email has both text and html content.

Reply:
If the email message is NOT multipart, we should probably use `email_message.get_body()` and relay that as whatever content_type it is.

Reply:
Filed https://github.com/mozilla/fx-private-relay/issues/377 for this.